centreon/tests/e2e/features/Platform-upgrade-update/02-platform-upgrade/index.ts

```
@@ -101,51 +101,81 @@ Given(
     cy.log(`Testing ${Cypress.env('IS_CLOUD') ? 'cloud' : 'onprem'} upgrade`);
 
-    return cy.getWebVersion().then(({ major_version }) => {
-      let major_version_from = '0';
+    return cy.getWebVersion().then(({ major_version, minor_version }) => {
+      let major_version_from = "0";
       switch (major_version_from_expression) {
-        case 'n - 1':
-          major_version_from = getCentreonPreviousMajorVersion(major_version);
+        case "n - 1":
+          const previousVersion =
+            getCentreonPreviousMajorVersion(major_version);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)